app/assets/stylesheets/layout_components/card.scss

```diff
@@ -0,0 +1,41 @@
+.card {
+  background: $white;
+  padding: $base-padding 0;
+  text-align: center;
+  margin-bottom: $base-margin;
+
+  &__title {
+    font-size: $base-font-size*1.45;
+    padding: 7px;
+    font-weight: bold;
+    text-align: center;
+
+    @include breakpoint(small only) {
+      font-size: $base-font-size*1.25;
+    }
+  }
+
+  &--blue-bg {
```

Comment:
I think this modifier should go on the title element - otherwise I would assume the whole card would have a blue background?

Reply:
you wouldn't be modifying a modifier, you'd be modifying an element - so you would have card__title--blue-bg. it reads better semantically - as it's clearer it's modifying the title and not the whole card (we do similar things with section__heading--grey-bg, etc.).

So in your example you would have:
<\div class="card">
  <\div class="card__title--blue-bg">
    <\h2>Waffles</h2>
  <\/div>
<\/div>

Reply:
okay, got it!